### ISSUE TITLE
Make uncrafting table buttons compatible with the mousewheel

### ISSUE THREAD

**Issue by EvergreenSystems** [TeamTwilight/twilightforest#1416]
### The Suggestion

Allow for players to use the mousewheel to scroll through uncrafting recipes instead of/in conjunction with clicking the buttons

### Why Should It Be in the Mod?

Some uncrafting recipes have _lots_ of potential iterations, for example things that use logs in their recipe (smoker), and if a specific block is desired sometimes it takes  a long time to get to the correct recipe. This gets painful when attempting to uncraft things en-masse, such as transforming logs from one type to another with the smoker. 

Basically a QoL upgrade

### Please Read and Confirm the Following

- [X] I have confirmed this suggestion is unique and has not been suggested already.
